### ISSUE TITLE
use SeqNet without SeqMatch

### ISSUE THREAD

**Issue by BIT-MJY** [oravus/seqNet#10]
Hello @oravus , 

Thanks for your fantastic job providing a nice way to fuse information from multiple frames. Now I only use SeqNet (Conv+SAP+L2-Norm) without SeqMatch, trained with my own dataset for LIDAR-based place recognition. However, I found SeqNet did not work well once the seq_len is small (< 20) and was hard to train to get a strong model. Thus I guess that whether SeqNet without SeqMatch works well largely depends on the distribution of training set and test set, or it largely depends on the output module of the raw descriptor generation algorithm. Is this right? Have you met some dataset where SeqNet can not work well? 

Best wishes!

**Comment by BIT-MJY** [comment]
Sorry for resubmitting
